RecoTracker/MkFit/plugins/MkFitSeedConverter.cc

```diff
@@ -147,7 +150,7 @@ mkfit::TrackVec MkFitSeedConverter::convertSeeds(const edm::View<TrajectorySeed>
         LogTrace("MkFitSeedConverter") << " adding hit detid " << detId.rawId() << " index " << clusterRef.index()
                                        << " ilay " << ilay;
         ret.back().addHitIdx(clusterRef.index(), ilay, 0);  // per-hit chi2 is not known
-      } else {
+      } else if (!(isPhase2_)) {
```

Comment:
This will have to be investigated further: as is, I'm unsure. For sure, the following doesn't work for phase-2, so a condition/protection is necessary.